core/client/fs/src/main/java/alluxio/client/file/MetadataCachingBaseFileSystem.java

```diff
@@ -228,4 +263,64 @@ public synchronized void close() throws IOException {
       super.close();
     }
   }
+
+  /**
+   * @param path the path need to drop metadata cache
+   * @return a mock RUIStatus object
+   */
+  private URIStatus dropMetadataCache(String path, String arg) {
```

Comment:
Can we keep MetadataCachingBaseFileSystem clean and move all the logics to Fuse?
here we only need

dropMetadataCache(AlluxioURI uri)
dropAllMetadataCache()
getMetadataCacheSize()

all the other uri processing should be done in the FUSE side to keep this class as clean as possible